Source/ResponseSerialization.swift

```diff
@@ -214,7 +226,7 @@ extension DataRequest {
         -> Self {
         appendResponseSerializer {
             // Start work that should be on the serialization queue.
-            let start = CFAbsoluteTimeGetCurrent()
+            let start = Date().timeIntervalSinceReferenceDate
```

Comment:
If we need more precision than this allows, we’ll have to find an alternative for `CFAbsoluteTimeGetCurrent` on Linux.

Reply:
[This thread](https://forums.swift.org/t/recommended-way-to-measure-time-in-swift/33326) on the Swift Forums also provides some guidance.

Reply:
After reading that thread, I’ve replaced this call with `ProcessInfo.processInfo.systemUptime`, which seems to work. I doubt this code needs nanosecond precision.